src/modules/app/components/SearchDialog.tsx

```
@@ -0,0 +1,175 @@
+                  <div className="flex w-full items-center justify-between">
+                    <span className="font-medium truncate">{note.title}</span>
+                    <span className="ml-4 text-xs text-muted-foreground flex-shrink-0">
+                      {new Intl.DateTimeFormat("en-US", {
+                        month: "short",
+                        day: "numeric",
+                      }).format(new Date(note.updatedAt))}
+                    </span>
+                  </div>
+                  <p
+                    className="text-xs text-muted-foreground w-full truncate"
+                    dangerouslySetInnerHTML={{ __html: note.snippet }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)